casper/src/test/resources/PoSTest.rho

```diff
@@ -467,6 +470,42 @@ match (
                 }
               }
             }
+          } |
+
+          contract getPosBalance(retCh) = {
+            new accBalances, vaultsCh, posVaultBalanceCh in {
+              contract accBalances(@(_, _, vault), @acc, resultCh) = {
+                new balanceCh in {
+                  @vault!("balance", *balanceCh) |
+                  for (@balance <- balanceCh) {
+                    resultCh!(acc + balance)
+                  }
+                }
+              } |
+
+              @posVault!("balance", *posVaultBalanceCh) |
+              for (@posVaultBalance <- posVaultBalanceCh) {
+                @PoS!("getActiveValidatorVaults", *vaultsCh) |
+                for (@vaults <- vaultsCh) {
+                  @ListOps!("fold", vaults, posVaultBalance, *accBalances, *retCh)
+                }
+              }
+            }
+          } |
```

Comment:
I think that the logic of the getPoSBalance contract should be part of the PoS API

Reply:
@odeac The PoS vault will always appear empty from the outside. It's only used as a temporary source vault for transfers during `closeBlock`, so I don't think it should make it to the API.

Reply:
@odeac we can make it part of the PoS API once we need it there.

Reply:
ok